modules/windows/collect.go

```diff
@@ -35,7 +35,7 @@ const (
 	collectorNetFrameworkCLRSecurity        = "netframework_clrsecurity"
 )
 
-func (w *WMI) collect() (map[string]int64, error) {
+func (w *WINDOWS) collect() (map[string]int64, error) {
```

Comment:
Should be just `Windows`

Reply:
I updated documentation removing `monitoring`.

Reply:
I didn't mean documentation: `func (w *WINDOWS)` => `func (w *Windows)`

Reply:
I pushed another commit renaming it. I was using the same pattern we have in `netdata/netdata`.